template/server.template.dart

```diff
@@ -39,7 +38,7 @@ Future<void> serverMain(List<String> args) async {
       .addMiddleware(middlewares([]))
       .addHandler(serveApp);
 
-  app.get('/<anything|.*>', handleGet);
+  app.all('/<anything|.*>', handleGet);
```

Comment:
I'd rename `handleGet` to `handler` because it handles all http methods, not just get